diff --git a/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/AutoCompleteTextArea.js b/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/AutoCompleteTextArea.js
index 2eada59..cf14a5e 100644
--- a/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/AutoCompleteTextArea.js
+++ b/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/AutoCompleteTextArea.js
@@ -74,7 +74,7 @@ export var AutoCompleteTextArea = function AutoCompleteTextArea(props) {
         var _props$onPressEnter;
         if (open && options.length > 0) return;
         (_props$onPressEnter = props.onPressEnter) === null || _props$onPressEnter === void 0 || _props$onPressEnter.call(props, e);
-      }
+      },
     }))
   });
 };
diff --git a/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/index.js b/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/index.js
index 39c13d9..e26fc25 100644
--- a/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/index.js
+++ b/node_modules/@ant-design/pro-chat/es/ProChat/components/InputArea/index.js
@@ -130,7 +130,7 @@ export var ChatInputArea = function ChatInputArea(props) {
               send();
             }
           }
-        })), mobile ? null : /*#__PURE__*/_jsx(Button, {
+        })), /*#__PURE__*/_jsx(Button, {
           loading: isLoading,
           type: "text",
           className: styles.btn,
